### DISCUSSION TITLE
Virtual keyboard not function on android

### DISCUSSION THREAD

**Discussion by Picobyte-sudo** [ppy/osu#16032] [Q&A] [answered]
So Basically for recent versions of osu!lazer the virtual keyboard is unfunctional which is kinda annoying  because I can't  look up any beatmaps on osu!direct I used to play one of the older versions this was a non issue however with newer ver it just doesn't work. Just requesting the devs if they could fix this bug in an update or does anyone know a fix? My current virtual  eyboard  is the default  Samsung keyboard.

**Comment by bdach** [comment]
did you try 2021.1207.0? is keyboard still broken?

**Reply by Feodor0090** [reply]
I'm using Redmi 9T (Android 10), keyboard is still broken on both 1207 and 1211.

**Comment by frenzibyte** [comment, marked as answer]
Tracked under https://github.com/ppy/osu/issues/8264.

@Picobyte-sudo @Feodor0090 please apply what's asked in https://github.com/ppy/osu/issues/8264#issuecomment-992111232 and attach them in that thread.
